src/main/webapp/gast/lemma.jsp

```diff
@@ -30,22 +30,34 @@
 
         Set<Einzelbeleg> listEinzelbeleg = lemma.getEinzelbelege();
 
-        boolean throwException = true;
-
+        boolean throwIdNotPublicException = true;
+        boolean throwContainsInvalidStrException = true;
         for (Einzelbeleg eb : listEinzelbeleg) {
             if (eb.getQuelle() != null && eb.getQuelle().getZuVeroeffentlichen() == 1) {
-                throwException = false;
+                throwIdNotPublicException = false;
                 break;
             }
         }
 
-        if (throwException) {
+        if(!lemma.getMghLemma().contains("[???]")){
+            throwContainsInvalidStrException = false;
+        }
+
+        if (throwIdNotPublicException) {
             if (session.getAttribute("Sprache").equals("de")) {
                 throw new IdNotPublicException("Lemma ID M" + id + " ist nicht zu veröffentlichen");
             } else{
                 throw new IdNotFoundException("Lemma ID M" + String.valueOf(id) + " is not to be published");
             }
         }
+
+        if (throwContainsInvalidStrException) {
+            if (session.getAttribute("Sprache").equals("de")) {
+                throw new ContainsInvalidStrException("Lemma ID M" + id + " enthält ungültige Zeichenkette -> '[???]'");
+            } else{
+                throw new ContainsInvalidStrException("Lemma ID M" + String.valueOf(id) + " contains invalid string -> '[???]'");
```

Comment:
It's OK like this right now, but for the long perspective we should also store these kind of error messages in the database.